Week3/homework/step2-4.js

```diff
@@ -2,6 +2,10 @@
 
 function Dog() {
   // add your code here
+  this.name = 'kuchuk';
+  this.color = 'black and white';
+  this.food = 'meat';
+  this.numLegs = 4;
```

Comment:
This is a bit limiting. You've hard-coded 'kuchuk', 'black and white', 'meat', and 4, into the Dog constructor. That means any hound you create will always have these values.

The idea here is that Dog should take name, color, and numLegs as parameters (and also food if you'd like to add that too) - which you'd pass in when you create it. Like this;

const hound = new Dog('kuchuk', 'black and white', 'meat', 4);

Reply:
I've added parameters to the function.